src/storage/storage_common.h

```diff
@@ -390,10 +390,11 @@ typedef enum
   SIX_LOCK = 7,			/* Shared and intention exclusive lock */
   U_LOCK = 8,			/* Update lock */
   X_LOCK = 9,			/* Exclusive lock */
```

Comment:
Lock types are ordered by how restrictive they are, in ascending order. With this in mind, probably BU_LOCK should be placed before X_LOCK. What do you think?